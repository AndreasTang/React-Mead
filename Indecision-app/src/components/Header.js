import React from 'react'

const Header = (props) => (
    <div className="header">
        <div className="container">
            <h1 className="header__title">{props.title}</h1>
            {props.subtitle && <h2 className="header__subtitle">{props.subtitle}</h2>}
        </div>
    </div>
)

Header.defaultProps = {
    title: '午餐地點選擇器'
}

export default Header