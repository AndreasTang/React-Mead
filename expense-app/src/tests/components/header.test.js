import React from 'react'
import Header from '../../components/header'
import { shallow } from 'enzyme'

test('Testing should Header component render currectly', () => {
    const wrapper = shallow(<Header />)
    expect(wrapper).toMatchSnapshot()
//    const renderer = new ReactShallowRenderer()
//    renderer.render(<Header />)
//    expect(renderer.getRenderOutput()).toMatchSnapshot()
})