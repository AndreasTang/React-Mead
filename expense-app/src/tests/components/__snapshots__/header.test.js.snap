// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing should Header component render currectly 1`] = `
<header>
  <h1>
    Damge Calculater
  </h1>
  <p>
    <NavLink
      activeClassName="is-active"
      exact={true}
      to="/"
    >
      Home
    </NavLink>
  </p>
  <p>
    <NavLink
      activeClassName="is-active"
      to="/Add"
    >
      Add
    </NavLink>
  </p>
  <p>
    <NavLink
      activeClassName="is-active"
      to="/Help"
    >
      Help
    </NavLink>
  </p>
</header>
`;
